packages/plugin/package.json

```diff
@@ -16,23 +16,22 @@
     "prepack": "bob prepack"
   },
   "dependencies": {
-    "@graphql-tools/code-file-loader": "~6.3.0",
-    "@graphql-tools/graphql-file-loader": "~6.2.0",
-    "@graphql-tools/graphql-tag-pluck": "~6.5.0",
+    "@graphql-tools/code-file-loader": "7.0.0-alpha-c42e811d.0",
```

Comment:
Awesome! I'm not sure if we need to merge it with the new v7, because it's still moving fast. 
Maybe we can get a release for v6 of the loaders with the fixed that we need from there? @B2o5T  is it the comments fix only, or do we need something else?

Reply:
I think we can't fix actual `code-file-loader` and `graphql-tag-pluck` because there is some breaking changes between v6 and v7 like `loadSync` of `CodeFileLoader` returns `Source[] | null` in v7 and `Source | null` in v6.

Or `gqlPluckFromCodeStringSync` of `graphql-tag-pluck` returns `Source[]` in v7 and `string` in v6
I think we need to wait for release `graphql-tools` v7. What do you think @dotansimha?

PS At least we already have PR that fix this issue 😅.